test/test_data/icecast.py

```diff
@@ -21,4 +21,16 @@
         "product": "icecast",
         "version": "2.4.4",
     },
+    {
+        "url": "http://ftp.fr.debian.org/debian/pool/main/i/icecast2/",
+        "package_name": "icecast2_2.4.0-1.1+deb8u1_amd64.deb",
+        "product": "icecast",
+        "version": "2.4.0",
```

Comment:
```suggestion
        "version": "2.4.0",
        "other_products": [],
```

Reply:
Adding other_product data so the new test will work. Assuming all this behaves, I'll have to go fix a lot of PRs but I think that's easier than it would have been to fix the new test PR 30 times or whatever it would have been.